db/seeds.rb

```diff
@@ -0,0 +1,5 @@
+Greeting.create(message: "Welcome to my website!")
+Greeting.create(message: "Hey there!")
+Greeting.create(message: "Good day!")
+Greeting.create(message: "Greetings #1!")
+Greeting.create(message: "Greetings #2!")
```

Comment:
- [ ] When I ran `rails db:seed` command to generate random greetings, I got an `UnknownAttributeError` - unknown attribute 'message' for Greeting. Kindly work on the attribute name so the seed command runs fine and random greetings are generated for the front-end app to show them as expected 👍🏽

<img width="960" alt="Screenshot 2023-11-03 at 6 22 38 PM" src="https://github.com/tchasinga/hi-back-end-react/assets/50941074/e3e5267c-08a5-485c-84e5-d728fb1b999a">